### ISSUE TITLE
QEMU/Passthrough and the Hardware Survey

### ISSUE THREAD

**Issue by Hirato** [ValveSoftware/steam-for-linux#5359]
Hello,

I have recently taken to getting this PCI-passthrough thing going via OVMF/VFIO.
As chance would have it, steam of course wanted to run the hardware survey on this virtual machine instance, so some of the more important details sent were as follows:

	Computer Information:
		Manufacturer:  QEMU
		Model:  Standard PC (Q35 + ICH9, 2009)
		Form Factor: Desktop
		No Touch Input Detected

	Operating System Version:
		Windows 10 (64 bit)
		NTFS:  Supported
		Crypto Provider Codes:  Supported 311 0x0 0x0 0x0

	Miscellaneous:
		UI Language:  English
		Media Type:  Undetermined
		Total Hard Disk Space Available:  2129351 Mb
		Largest Free Hard Disk Block:  952524 Mb
		OS Install Date: Jan 01 1970
		Game Controller: None detected
		VR Headset: None detected

Basically I have a Linux host, with Windows running in a virtual machine.
So what I want to know is:

1. For the purposes of the hardware survey, was this counted as a Windows install?
2. And would it be possible for Steam to detect the hypervisor (like you do with wine at the moment) to collect statistics on the prevalence thereof, and provide a breakdown on the OS level of how many instances are virtualised?

I do personally think there might be some value in tracking how manyf OS installs are on the metal and how many are virtualised - though I expect the percentage of such users to be fairly small minority within the already small minority that is the Linux user base.

**Comment by Yepoleb** [comment]
If you're using a Windows VM you're just a regular Windows user to game developers. The hypervisor doesn't affect the game in any way so why should they track it?
